src/InvoiceGenerator/InvoiceGenerator.BusinessLogic/InvoiceGenerator.BusinessLogic.csproj

```diff
@@ -0,0 +1,13 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+	  <TargetFramework>netcoreapp6.0</TargetFramework>
+    <Nullable>enable</Nullable>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <PackageReference Include="Microsoft.EntityFrameworkCore" Version="6.0.5" />
```

Comment:
Once the DbContext is moved from here (into the Domain layer) we can remove these references and set the TargetFramework back to netstandard2.1